jscomp/others/js_json.res

```diff
@@ -34,13 +34,15 @@ type rec t =
   | Object(Js.Dict.t<t>)
   | Array(array<t>)
 
-type rec kind<_> =
-  | String: kind<Js_string.t>
-  | Number: kind<float>
-  | Object: kind<Js_dict.t<t>>
-  | Array: kind<array<t>>
-  | Boolean: kind<bool>
-  | Null: kind<Js_types.null_val>
+module Kind = {
+  type rec s<_> =
```

Comment:
s/ s / t

Reply:
Not sure I understand your notation, but

s / kind / t https://github.com/rescript-lang/rescript-compiler/pull/6317/commits/695bb47844118b3d26523ee61029f20c7c67b4e7